modules/toxiproxy/src/test/java/org/testcontainers/containers/ToxiproxyTest.java

```diff
@@ -192,6 +192,35 @@ public void testControlPort() {
         assertEquals("control port is mapped from port 8474", toxiproxy.getMappedPort(8474), controlPort);
     }
 
+    @Test
+    public void testResetProxy() throws IOException {
+        // obtainProxyObject {
+        final ToxiproxyContainer.ContainerProxy proxy = toxiproxy.getProxy(redis, 6379);
+        // }
+
+        // obtainProxiedHostAndPortForHostMachine {
+        final String ipAddressViaToxiproxy = proxy.getContainerIpAddress();
+        final int portViaToxiproxy = proxy.getProxyPort();
+        // }
+
+        final Jedis jedis = createJedis(ipAddressViaToxiproxy, portViaToxiproxy);
+        jedis.set("somekey", "somevalue");
+
+        // addingLatency {
+        proxy.toxics().latency("latency", ToxicDirection.DOWNSTREAM, 1_000);
+        // from now on the connection latency should be around 1000ms.
+        // }
+
+        checkCallWithLatency(jedis, "before resetting", 1_000, 1_050);
+
+        // reset {
+        toxiproxy.reset();
+        // latency should be back to normal
+        // }
```

Comment:
`inside_block` are used for documentation. See toxiproxy docs [here](https://github.com/testcontainers/testcontainers-java/blob/2d09155faa529074f53b10a2e7f828bf531efa2d/docs/modules/toxiproxy.md?plain=1). My suggestion is to update docs providing this block and delete the rest of them in this test and avoid breaking existing docs.

```suggestion
        // reset {
        toxiproxy.reset();
        // }
```

Reply:
Thanks for looking into it, I've updated the PR